doc/user/content/sql/create-source/postgres.md

```diff
@@ -112,7 +118,10 @@ Materialize does not support changes to schemas for existing publications, and w
 
 ##### Supported types
```

Comment:
@philip-stoev, addressed [your comments](https://github.com/MaterializeInc/materialize/pull/16010#pullrequestreview-1180536527) in this section.